reference/Upload-API.yaml

```diff
@@ -195,6 +195,10 @@ paths:
               description: ''
               type: object
               properties:
+                filename:
+                  type: string
+                  minLength: 1
```

Comment:
i'd be tempted to make this slightly longer

Reply:
we have no such validation in lambda, so it would be inconsistent with it... but maybe worth modifying it in the api itself later